src/app/add-item/add-item.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-add-item',
+  templateUrl: './add-item.component.html',
+  styleUrls: ['./add-item.component.scss'],
+})
+
+export class AddItemComponent implements OnInit {
+  items = [];
+  // itemList -> chenge next whit the list of lists for choice one
+  itemList = 'example';
+  itemID: number = this.items.length + 1;
+  itemTitle: string;
+  itemDescription: string;
+
+  ngOnInit() {
+    this.itemTitle = '';
```

Comment:
you can put this in field initialization, ngOnInit is used for initializations that requires the component to be ready.